ui/src/dashboards/components/QueryMaker.tsx

```diff
@@ -63,7 +63,7 @@ const QueryMaker: SFC<Props> = ({
           query={buildText(activeQuery)}
           config={activeQuery}
           onUpdate={rawTextBinder(
-            source.links,
+            source,
```

Comment:
was this broken? how this could be changed without refactoring `rawTextBinder` to work from a source instead of just links? i haven't looked at the original `rawTextBinder` implementation but there aren't changes to it in this PR

Reply:
@jaredscheib 
This was updated because the `actions.editRawTextAsync` was uses `source` when calling `replaceTempVars` above. 

The `rawTextBinder` was updated above on line `16`

Reply:
ah i see, at a glance it looked like just an interface definition. why is rawTextBinder in this file? seems like a util

Reply:
True,  I can move it.